packages/devtools_app/lib/src/shared/scaffold.dart

```diff
@@ -556,16 +556,36 @@ class KeyboardShortcuts extends StatelessWidget {
   final ShortcutsConfiguration keyboardShortcuts;
   final Widget child;
 
+  @override
+  KeyboardShortcutsState createState() => KeyboardShortcutsState();
+}
+
+class KeyboardShortcutsState extends State<KeyboardShortcuts>
+    with AutoDisposeMixin {
+  FocusNode _focusNode;
+
+  @override
+  void initState() {
+    super.initState();
+    _focusNode = FocusNode(debugLabel: 'keyboard-shortcuts');
+    autoDisposeFocusNode(_focusNode);
+  }
+
   @override
   Widget build(BuildContext context) {
-    if (keyboardShortcuts.isEmpty) {
-      return child;
+    if (widget.keyboardShortcuts.isEmpty) {
+      return widget.child;
     }
-    return Shortcuts(
-      shortcuts: keyboardShortcuts.shortcuts,
-      child: Actions(
-        actions: keyboardShortcuts.actions,
-        child: child,
+
+    return GestureDetector(
+      behavior: HitTestBehavior.opaque,
+      onTap: () => FocusScope.of(context).requestFocus(_focusNode),
```

Comment:
This is a little concerning since KeyboardShortcuts is a top level wrapper widget around the scaffold. If a FocusNode further down the tree requests focus, will those requests take precedence over this one? I would think so because this GestureDetector would be "behind" or further up the tree than other gesture detectors down the tree, but just want to confirm.

Reply:
Yeah, I've added some logging in `FocusManager` to check, it seems like everything that already had a `FocusNode` is still getting focus as we would want, and this new focus node only gets focus when something that doesn't have a focus node in scope is tapped.